CRM/Iats/Upgrader.php

```diff
@@ -13,8 +13,8 @@ public function getCurrentRevision() {
     try {
       $xmlfile = CRM_Core_Resources::singleton()->getPath('com.iatspayments.civicrm','info.xml');
       $myxml = simplexml_load_file($xmlfile);
-      $version = (string)$myxml->version;
-      CRM_Core_BAO_Setting::setItem($version, 'iATS Payments Extension', 'iats_extension_version');
+      $version = (string) $myxml->version;
+      Civi::settings()->set('iats_extension_version', $version);
```

Comment:
Is that a syntax fix @seamuslee001 ?

Reply:
Yes there is usually meant to be a space between the casting and variable AFAIK